rhc-ose-ansible/roles/secure-registry/defaults/main.yaml

```diff
@@ -0,0 +1,12 @@
+openshift_registry_project: "default"
+docker_registry_dc_name: "docker-registry"
+registry_certificate: ""
+registry_key: ""
+registry_ca: "/etc/origin/master/ca.crt"
+registry_serviceaccount: "registry"
+registry-secret-name: "registry-secret"
+openshift: 
```

Comment:
Was having trouble getting the openshift-ansible `openshift_common` role to run successfully to set the variables for the openshift binaries. This is a placeholder till i can get that up and running.